projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/deploy/JobImageDeployer.java

```diff
@@ -312,17 +307,14 @@ private void updateCronJob(DataJob dataJob, JobDeployment jobDeployment, String
   }
 
   private String getJobVdkImage(JobDeployment jobDeployment) {
```

Comment:
Does this method duplicate the SupportedPythonVersions.getVdkImage()?

Reply:
Good point. With the cleanup, this one is not really needed anymore. Switched to use the `.getVdkImage()` in supportedPythonVersions.
Thanks for the review